oas_docs/output/kibana.yaml

```diff
@@ -52682,6 +52682,22 @@ paths:
             oneOf:
               - type: string
               - type: array
+        - description: |
+            Specifies whether to apply logical AND filtering for specific fields. Accepts either a string with values "tags" or "locations" or an array containing both.
+          in: query
+          name: useLogicalAndFor
```

Comment:
i wonder if we can improve this API, instead of using param useLogicalAndFor
possibly tagsOperation: "AND" | "OR" might be more readable?
same goes for location

locationsOperator: "AND" | "OR"

Reply:
I implemented like this to avoid having a different query parameter for each field we want to enable the logical AND for. 

Implementing it like this makes very easy to extend it for a new field (see [this commit](https://github.com/elastic/kibana/pull/217985/commits/4a3aa5a371230dd4533877ce1c986d9321d7d195) to enable the logical AND for locations.